app/views/shared/_off_canvas_menu.html.erb

```diff
@@ -17,7 +17,6 @@
     data-transition-leave="transition-opacity ease-linear duration-200"
     data-transition-leave-start="transform opacity-100"
     data-transition-leave-end="transform opacity-0">
-  </div>
```

Comment:
There is a chance that this is the wrong `</div>` closing tag I removed, but there is at least one redundant `</div>` in this template.